lib/broccoli_sass_compiler.js

```diff
@@ -667,9 +671,16 @@ module.exports = class BroccoliSassCompiler extends BroccoliPlugin {
   }
 
   handleSuccess(details, result) {
-    mkdirp.sync(path.dirname(details.fullCssFilename));
-    fs.writeFileSync(details.fullCssFilename, result.css);
+    let cachedFile = path.join(this.cachePath, details.sassFilename);
+    let outputFile = details.fullCssFilename;
 
+    mkdirp.sync(path.dirname(cachedFile));
+    fs.writeFileSync(cachedFile, result.css);
+
+    mkdirp.sync(path.dirname(outputFile));
+    ensureSymlink(cachedFile, outputFile);
+
+    return this.events.emit("compiled", details, result);
     return this.events.emit("compiled", details, result);
```

Comment:
is it intentional to send the `compiled` event twice?

Reply:
good catch, it is not.